File: help/pt_pt/exercicios_da_ficha_3/Ficha03_007/Ficha03_007.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Teorema de Pitágoras\n" );
    document.write("Cateto a \t: " );
    document.write();// print text
    a = parseFloat(prompt("a","0.0") );
    
    document.write(a + "\n");// print input 
    document.write("Cateto b \t: " );
    document.write();// print text
    b = parseFloat(prompt("b","0.0") );
    
    document.write(b + "\n");// print input 
    var c = Math.sqrt( Math.pow( a , 2 ) + Math.pow( b , 2 ) ) ;
    document.write("Hipotnusa \t: " + c );
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 14:29

File: help/pt_pt/exercicios_da_ficha_3/Ficha03_007/Ficha03_007.c
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
//Main Function
int main() {
    printf("Teorema de Pitágoras\n");
    printf("Cateto a \t: ");
    float a;
    scanf("%f", &a);
    printf("Cateto b \t: ");
    float b;
    scanf("%f", &b);
    float c = sqrt( pow( a , 2 ) + pow( b , 2 ) ) ;
    printf("Hipotnusa \t: %f",c );
    return 0;
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 14:29
